<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scavenger Hunt Logger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QComboBox" name="comboEvent"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="TaskView" name="taskView"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="buttonRemove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="buttonAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="buttonRename">
      <property name="text">
       <string>Rename</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QComboBox" name="comboTeam"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEditor"/>
   <addaction name="actionTasks"/>
   <addaction name="actionRankings"/>
   <addaction name="actionSettings"/>
   <addaction name="actionConsole"/>
  </widget>
  <action name="actionEditor">
   <property name="text">
    <string>Teams</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="text">
    <string>Tasks</string>
   </property>
  </action>
  <action name="actionRankings">
   <property name="text">
    <string>Rankings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskView</class>
   <extends>QListView</extends>
   <header>taskview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
